src/Stratis.Bitcoin.Features.BlockStore.Tests/BlockStoreTests.cs

```diff
@@ -345,7 +345,6 @@ public async Task ReorgedBlocksAreDeletedFromRepositoryIfReorgDetectedAsync()
             this.chainState.IsAtBestChainTip = false;
 
             // Make sure only longest chain is saved.
-            Assert.Equal(1, this.repositorySavesCount);
```

Comment:
Is this removed on purpose?

Reply:
Yes. 
Now we don't save when we are at chain tip but when we are withing 5 blocks from it.